CRM/Core/Menu.php

```diff
@@ -216,7 +216,7 @@ public static function fillMenuValues(&$menu, $path) {
     );
     $fieldsPresent = array();
     foreach ($fieldsToPropagate as $field) {
-      $fieldsPresent[$field] = CRM_Utils_Array::value($field, $menu[$path]) !== NULL ? TRUE : FALSE;
+      $fieldsPresent[$field] = isset($menu[$path][$field]);
```

Comment:
Since the old bit here is longer the other cleaned expressions, I gave this a slightly closer look and some REPL testing - looks good.